controllers/user.js

```diff
@@ -0,0 +1,26 @@
+const User = require("../models/User");
+const jwt = require('jsonwebtoken');
+
+
+module.exports.login = async function (request, response) {
+    const user = await User.findOne({email: request.body.email})
+    if (user) {
+        if (request.body.password === user.password) {
```

Comment:
are you going to [store](https://stackoverflow.com/questions/1054022/best-way-to-store-password-in-database) passwords as is ? :)